### PR TITLE
test: Add d3d12 native test

### PR DESCRIPTION
**Pull request by karasusan**
This change is for corresponding the native test of d3d12 graphics API.

### CODE REVIEW

**Comment by sindharta** [comment]
d3d12? not opengl ?

File: Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace webrtc = ::webrtc;
 
 #define DefPtr(_a) _COM_SMARTPTR_TYPEDEF(_a, __uuidof(_a))
 DefPtr(ID3D12CommandAllocator);
-DefPtr(ID3D12GraphicsCommandList4);
+DefPtr(ID3D12GraphicsCommandList);
```
Comment (sindharta) [inline Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h:16]: I think we should use the newer interface. What is the reason of this change?

File: Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h [hunk 2]
```diff
@@ -74,7 +74,7 @@ class D3D12GraphicsDevice : public IGraphicsDevice{
 
     //[TODO-sin: 2019-12-2] //This should be allocated for each frame.
     ID3D12CommandAllocatorPtr m_commandAllocator;
-    ID3D12GraphicsCommandList4Ptr m_commandList;
+    ID3D12GraphicsCommandListPtr m_commandList;
```
Comment (sindharta) [inline Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h:77]: I think we should use the newer interface. What is the reason of this change?

File: Plugin~/WebRTCPluginTest/GraphicsDeviceTestBase.cpp [hunk 3]
```diff
@@ -33,7 +36,7 @@ Microsoft::WRL::ComPtr<ID3D11DeviceContext> pD3D11DeviceContext;
 
 Microsoft::WRL::ComPtr<IDXGIAdapter1> pAdapter1;
 Microsoft::WRL::ComPtr<IDXGIFactory4> pFactory4;
-Microsoft::WRL::ComPtr<ID3D12Device5> pD3D12Device;
+Microsoft::WRL::ComPtr<ID3D12Device> pD3D12Device;
```
Comment (sindharta) [inline Plugin~/WebRTCPluginTest/GraphicsDeviceTestBase.cpp:39]: I think we should use the newer interface. What is the reason of this change?

File: Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h (hunk 1 above)
Comment (karasusan) [reply Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h:16]: OK, I will revert this and test.

Comment (karasusan) [reply Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h]: Mmm, this changes is passed test on local PC but failed on Yamato.
Graphics driver on Bokken image is already updated. Do you have any idea?

Comment (sindharta) [reply Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h]: Hm, try getting the version of DX12 on both local PC and Yamato and make sure they are the same ?
https://support.microsoft.com/en-us/help/179113/how-to-install-the-latest-version-of-directx

Comment (karasusan) [reply Plugin~/WebRTCPlugin/GraphicsDevice/D3D12/D3D12GraphicsDevice.h]: I got it!
The windows version is earlier than required version for test.
I am upgrading the Windows OS of the testing environment.
